exampleCourse/questions/demo/drawing/resistorCapacitorCircuit/question.html

```diff
@@ -0,0 +1,32 @@
+<pl-question-panel>
+
+    <p>
+        <pl-drawing width="440" height="260"  grid-size="0" answers-name="answer">
+            <pl-drawing-initial>
+                <pl-battery        x1={{params.pA.0}} y1={{params.pA.1}} x2={{params.pB.0}} y2={{params.pB.1}} label="{{params.V}} V"></pl-battery>
+                <pl-resistor        x1={{params.pB.0}} y1={{params.pB.1}} x2={{params.pC.0}} y2={{params.pC.1}} label="{{params.R1}}\\Omega"></pl-resistor>
```

Comment:
As a best practice, attribute values should be enclosed in double quotes. It's not strictly necessary in this case, but it is needed if there are spaces, quotes, or angle brackets in the string, so we usually include them so folks don't have to think too hard if they're pattern-matching against our example questions.

Reply:
got it!